schemas/components/schemas/VpnResult.yaml

```
@@ -46,6 +46,12 @@ properties:
       - publicVPN
       - auxiliaryMobile
       - osMismatch
+  confidence:
+    type: string
```

Feels like a good place to use enum? 
```suggestion
    type: string
    enum: [low, medium, high]
```